az-core/src/main/java/azkaban/Constants.java

```diff
@@ -692,6 +692,9 @@ public static class ContainerizedDispatchManagerProperties {
     public static final String KUBERNETES_POD_SERVICE_ACCOUNT_TOKEN_AUTOMOUNT =
         KUBERNETES_POD_PREFIX + "service.account.token.automount";
 
+    public static final String KUBERNETES_POD_SERVICE_LINKS =
+        KUBERNETES_POD_PREFIX + "service.account.links";
```

Comment:
Consider renaming to something like `inject.service.links` ?
This is not service account related and could be confusing.

Reply:
Will just change to service links. thanks!